build-release

```diff
@@ -5,12 +5,20 @@
 # ARCHITECTURE: architecture for the build
 # ISSUE_NUMBER: the number of the issue that triggered the build
 # DRY_RUN_PARAM: parameter that to use dry-run
+# ONLY_BUILD_TOOLCONF: if True, tells me to only build cmssw-tool-conf 
 
 HERE=$WORKSPACE/build
 BUILD_DIR=$HERE/$CMSSW_X_Y_Z-build
 mkdir -p $BUILD_DIR
 QUEUE=$CMSSW_QUEUE
 
+if [ "X$ONLY_BUILD_TOOLCONF" = Xtrue ]; then
+  POST_ACTION='TOOL_CONF_BUILDING'
+else
+  POST_ACTION='BUILDING'
+fi
+
+
 git clone git@github.com:cms-sw/cmsdist.git $BUILD_DIR/CMSDIST
```

Comment:
It deletes the workspace before the build starts. And also it can choose a different machine for the build. Jenkins can assign cmsbuild13 to build tool-conf and then cmsbuil15 to build cmssw.